### PR TITLE
[16.0] [MIG] l10n it location nuts 

### PR DESCRIPTION
**Pull request by Vincenzo9191**
Migration l10n_it_location_nuts from 14.0 to 16.0
Depends on PR OCA/partner-contact#1398

### CODE REVIEW

**Comment by SirTakobi** [comment]
> Migration l10n_it_location_nuts from 14.0 to 16.0
Depends on PR https://github.com/OCA/partner-contact/pull/1398

Per includere https://github.com/OCA/partner-contact/pull/1398 puoi aggiungere una riga a `test-requirements.txt` (vedi ad esempio https://github.com/OCA/l10n-italy/pull/2992/commits/383ff8724c449abc9655d2ad2f722aa8915e6226) così verranno eseguiti i test e runboat

**Comment by primes2h** [comment]
Ciao, grazie della PR.
Un cherry-pick di questo commit https://github.com/OCA/l10n-italy/pull/3041/commits/5e860ae341bdf91e852c12193a47fd2c701c8e1d sarebbe ottimo, così restiamo allineati alla 14.0.

**Comment by primes2h** [comment]
> Migration l10n_it_location_nuts from 14.0 to 16.0 Depends on PR [OCA/partner-contact#1398](https://github.com/OCA/partner-contact/pull/1398)

Dimenticavo, andrebbe aggiornata allo stesso modo anche la PR qui sopra con questo commit https://github.com/OCA/partner-contact/pull/1400/commits/5414bbc92d11a923739c41709bd09bbb881a4a2f

File: l10n_it_location_nuts/hooks.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def post_init_hook(cr, registry):
+    """Define Italian specific configuration in res.country."""
+    with api.Environment.manage():
```
Comment (TonyMasciI) [inline l10n_it_location_nuts/hooks.py:13]: `manage()` Since Odoo 15.0, Environment.manage() is useless.

**Review by TonyMasciI — CHANGES_REQUESTED** [review]
Revisione tecnica:

- Modulo installato senza problemi
- Test unitari OK

**Review by Borruso — CHANGES_REQUESTED** [review]
[[FIX] category in manifest file ...](https://github.com/OCA/l10n-italy/pull/3029/commits/0045f91c8aa003c8631c42ab39bf4d80ab8184ab)

questo commit riesci a mergiarlo per con il commit di migrazione

File: l10n_it_location_nuts/tests/test_nuts.py [hunk 2]
```diff
@@ -2,10 +2,10 @@
 # Copyright 2022 Simone Rubino - TAKOBI
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo.tests.common import SavepointCase
```
Comment (Borruso) [inline l10n_it_location_nuts/tests/test_nuts.py]: questa parte dovrebbe essere nel commit di migrazione [[MIG] l10n_it_location_nuts: Migration to 16.0](https://github.com/OCA/l10n-italy/pull/3029/commits/f7243bfb83ae0b648041c65a60e25a88e3abafef)

**Pull request closed without merging**